lib/bolt/config.rb

```diff
@@ -402,5 +467,12 @@ def casefold(path)
         l =~ /[A-Za-z]/ ? "[#{l.upcase}#{l.downcase}]" : l
       end.join
     end
+
+    def config_loaded
+      msg = <<~MSG
+        Loaded configuration from: '#{[*@config_files, boltdir.config_file].join("', '")}'
```

Comment:
This can just be `config_files` now, since that will contain the Boltdir config path as well.

Reply:
It also might make more sense for this to just be inline in the CLI class where it's used instead of being a method on the `Config` object.